libs/gamepad/src/GamepadHeaders/GamepadStruct.h

```diff
@@ -17,6 +19,6 @@ namespace gamepad {
     float leftStickMagnitude;
     float leftTriggerMagnitude;
     float rightTriggerMagnitude;
-    float pressDuration[32];
+    std::array<float, 32> pressDuration;
```

Comment:
oh. I'm scared of this one. I read something about the spec doesn't require the implementation to be just a C array. But this struct definitely needs to be:
```
...
float
float
float_0
float_1
...
float_31
```

Let's see what happens.

Reply:
lmao. I could've built it locally first. But it works! Quick POC:

```cpp
if (pressing(input, Buttons[Button::R3])) {
  printf("Pressing R3 for %f seconds\n", input.pressDuration[Button::R3]);
}
```

```text
Pressing R3 for 3.654727 seconds
Pressing R3 for 3.671431 seconds
Pressing R3 for 3.725321 seconds
Pressing R3 for 3.741988 seconds
Pressing R3 for 3.780785 seconds
Pressing R3 for 3.829443 seconds
```